pkg/ebpf/tracee.go

```diff
@@ -66,11 +66,11 @@ type Tracee struct {
 	stats     metrics.Stats
 	sigEngine *engine.Engine
 	// Events
-	eventsSorter     *sorting.EventsChronologicalSorter
-	eventsPool       *sync.Pool
-	eventsParamTypes map[events.ID][]bufferdecoder.ArgType
-	eventProcessor   map[events.ID][]func(evt *trace.Event) error
-	eventDerivations derive.Table
+	eventsSorter       *sorting.EventsChronologicalSorter
+	eventsPool         *sync.Pool
+	eventArgumentTypes map[events.ID][]trace.DecodeAs
```

Comment:
Arguments/Parameters should be replaced by data fields, which is the terminology used by the new event structure

Reply:
Renaming to `eventDataDecodeTypes` is ok for you?